ui/app/styles/core/label.scss

```diff
@@ -20,16 +20,6 @@
   &:not(:last-child) {
     margin-bottom: 0.25rem;
   }
-  // Sizes
-  &.is-small {
```

Comment:
Out of curiosity - is the final step going through the hbs files to update the css accordingly?